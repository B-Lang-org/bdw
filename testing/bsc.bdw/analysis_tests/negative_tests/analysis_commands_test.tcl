# Opens the specified project
handlers::open_project project.bspec

handlers::load_package

handlers::load_package Mult5

handlers::import_hierarchy

handlers::import_hierarchy Mult4Tb

handlers::reload_packages

handlers::add_type

handlers::load_module

handlers::load_module mkMult5

handlers::reload_module

handlers::show_schedule

handlers::show_schedule mkMult5

# Closes the project
handlers::close_project

# Exits the workstation
exit
