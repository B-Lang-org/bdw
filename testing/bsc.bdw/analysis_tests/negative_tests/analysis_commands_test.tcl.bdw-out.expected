The package name is not specified
The package 'Mult5' does not exist.
There are no loaded packages.
There are no loaded packages.
There are no loaded packages.
The type is not specified
The module name is not specified
The module 'mkMult5' does not exist.
There is no module loaded.
There is no module loaded.
The module 'mkMult5' does not exist.
